stopcovid/dialog/engine.py

```diff
@@ -110,23 +105,13 @@ def execute(
             )
             return []
 
-        if self.drill_instance_id:
-            return [
-                DrillStarted(
-                    phone_number=self.phone_number,
-                    user_profile=dialog_state.user_profile,
-                    drill=self.drill,
-                    first_prompt=self.drill.first_prompt(),
-                    drill_instance_id=self.drill_instance_id,
-                )
-            ]
-        # TODO: remove return once scadmin is creating DrillInstances
         return [
             DrillStarted(
```

Comment:
I think we can also remove the default factory on `DrillStarted.drill_instance_id`